headers/types/common/common.h

```diff
@@ -1230,4 +1230,83 @@ struct monster_sprite_data_entry {
 };
 ASSERT_SIZE(struct monster_sprite_data_entry, 2);
 
+// Holds basic information about a monster.
+struct monster_data_table_entry {
+    uint16_t entity_id; // 0x0: A entity-unique ID to identify the entity in both of its 2 entries.
+                        // (Independent from file index)
+    uint16_t
+        unk31; // 0x2: Unknown. Pokemon with the same category string have the same value in here.
```

Comment:
Why is this unk 31?

I usually prefer to name things by their hex offset, e.g. `unk_0x2`. That way if we later figure out what one is, we won't have to change/leave gaps in the numbering of all unknown fields that come after it.